data/json/items/melee.json

```diff
@@ -3805,6 +3805,31 @@
     "use_action": { "type": "knife" },
     "category" : "weapons",
     "qualities": [["CUT", 1], ["BUTCHER", 8]]
+    },{
+    "id": "scimitar",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "scimitar",
+    "description": "This is a curved sword associated with various middle eastern and central asian countries. Designed for slashing, and quite deadly against unarmored targets.",
```

Comment:
Please capitalize regions: "Middle Eastern and Central Asian countries" and put double space after a period. (It wasn't fixed in initial implementation but we have a second chance).